pkgs/servers/monitoring/prometheus/dcgm-exporter/default.nix

```
@@ -0,0 +1,67 @@
+{ lib
+, buildGoModule
+, addOpenGLRunpath
+, fetchFromGitHub
+, dcgm
+, linuxPackages
+}:
+buildGoModule rec {
+  pname = "dcgm-exporter";
+  version = "3.1.8-3.1.5";
+
+  src = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "dcgm-exporter";
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```
being more explicit is preferred when specifying revs